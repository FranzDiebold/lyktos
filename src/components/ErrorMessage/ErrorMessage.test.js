import React from 'react';
import renderer from 'react-test-renderer';

import ErrorMessage from './ErrorMessage';

it('renders error message correctly', () => {
    const tree = renderer
        .create(<ErrorMessage message="This is a test error message." />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
