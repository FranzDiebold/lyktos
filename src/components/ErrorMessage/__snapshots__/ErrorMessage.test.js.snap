// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error message correctly 1`] = `
<article
  className="message is-warning"
>
  <div
    className="message-header"
  >
    <p>
      Error
    </p>
  </div>
  <div
    className="message-body"
  >
    <div
      className="columns"
    >
      <div
        className="column"
      >
        This is a test error message.
      </div>
    </div>
    <div
      className="columns"
    >
      <div
        className="column has-text-centered"
      >
        <button
          className="button"
          onClick={[Function]}
          type="button"
        >
          <span
            className="icon"
          >
            <svg
              style={
                Object {
                  "height": 24,
                  "width": 24,
                }
              }
              viewBox="0 0 24 24"
            >
              <path
                d="M19,12H22.32L17.37,16.95L12.42,12H16.97C17,10.46 16.42,8.93 15.24,7.75C12.9,5.41 9.1,5.41 6.76,7.75C4.42,10.09 4.42,13.9 6.76,16.24C8.6,18.08 11.36,18.47 13.58,17.41L15.05,18.88C12,20.69 8,20.29 5.34,17.65C2.22,14.53 2.23,9.47 5.35,6.35C8.5,3.22 13.53,3.21 16.66,6.34C18.22,7.9 19,9.95 19,12Z"
                fill="currentColor"
              />
            </svg>
          </span>
          <span>
            Reload
          </span>
        </button>
      </div>
    </div>
  </div>
</article>
`;
